parsely-amp-tracking.php

```diff
@@ -0,0 +1,25 @@
+add_action( 'pre_amp_render_post', 'parsely_amp_add_actions' );
```

Comment:
We'll actually want to get rid of this as a separate file and instead insert this code into `wp-parsely.php`, as part of the `Parsely` class. 

To start, let's make this line `add_action('pre_amp_render_post', array($this, 'parsely_add_amp_actions'));` and add it beneath [this line](https://github.com/Parsely/wp-parsely/blob/release/v1.10/wp-parsely.php#L85) in the `Parsely` class's `__construct` function. We'll need to pass that second callback as an array with `$this` since we'll be including the functions below within the `Parsely` class